src/address_operands.rs

```diff
@@ -119,7 +119,7 @@ fn reg_and_imm_write(
             let dst0 = vm.get_register(opcode.dst0_index);
             let offset = opcode.imm1;
             let res = dst0 + TaggedValue::new_raw_integer(U256::from(offset));
-            vm.set_register(opcode.dst0_index, res);
+            //vm.set_register(opcode.dst0_index, res);
```

Comment:
When storing something this way:
`add r1,r2,stack[r3+5]`
It would still store the result on register r3